Ms 1850 system doc (#57)
Meshcloud/meshcloud-docs: docs/meshstack.authorization.md

Cloud Foundry uses the Identity broker only to authenticate users. Authorization is done as described before via roles set by meshStack.

#### AWS

The Project User Roles are mapped to AWS IAM Roles, which are the authorization roles in AWS. The access rights associated with these IAM roles is not part of meshStack.

## Service User

[Service Users](meshcloud.service-user.md) are technical users, that can be created per platform instance in a project. They are mainly used in CI/CD pipelines. They are local platform users and can therefore only be used to access a specific project in a specific cloud platform. The password of such a generated user is only downloaded once when a service user is created. It is a random generated, 24 characters long password generated with Java's SecureRandom class. meshStack does not store this password. It is the user’s responsibility to safely store it. If the password is somehow compromised, the service user can easily be deleted and replaced by a new service user.

> Service Users are available for Cloud Foundry and OpenStack cloud platforms. Other cloud platforms supported by meshStack like AWS or Kubernetes offer superior “native” alternatives to service users (i.e. IAM Policies & Credentials, ServiceAccounts).

Reviewer: Maybe directly add a link to the external docs for IAM Policies to AWS or Docker?

Author: added link to AWS IAM docs

> A Service User can be created and deleted by all users assigned to the project. Information about which user created the Service User and therefore is responsible for the user is available in meshStack.

Reviewer: ```suggestion
A Service User can be created and deleted by all users assigned to the project. Information about the Service User creator is available in meshStack.
```
Otherwise sounds a bit clunky.

Author: I split it up into two sentences. I think mentioning, that the creator is responsible for the secure usage of teh service user is a good remark.

## User Revocation

> User Revocation on [project](meshcloud.project.md#unassign-user-from-a-meshproject) and [customer](meshcloud.customer.md#remove-users-from-a-meshcustomer) level allow Customer Admins to always limit access to the meshCustomer and meshProjects to the users that actually need access. Users who no longer should have access can easily be revoked access. Administrators also have the possibility to revoke access of a user to all meshCustomers and meshProjects and deactivate this user completely in the complete meshStack via the [delete user](administration.users.md#delete-user) functionality.

Reviewer: ```suggestion
User Revocation on [project](meshcloud.project.md#unassign-user-from-a-meshproject) and [customer](meshcloud.customer.md#remove-users-from-a-meshcustomer) level allow Customer Admins to always limit access to the meshCustomer and meshProjects to the users that actually needs it. Access rights can also easily be revoked here. Administrators also have the possibility to revoke access of a user to all meshCustomers and meshProjects and deactivate this user completely in the whole meshStack via the [delete user](administration.users.md#delete-user) functionality.
```

Author: somehow github did not show the actual diff. I think i found all relevant parts and integrated them
